lib/metadata.ts

```diff
@@ -71,6 +71,7 @@ export const createMetadata: MetadataGenerator = (
     twitter: {
       card: 'summary_large_image',
       creator: twitterHandle,
+      site: twitterHandle,
```

Comment:
What does this do? I'm not really sure where twitterHandle gets populated. Is it a prismic thing?

Ideally we'd say X handle, but this is fine

Reply:
@chrisgervang Actually, sir, I use a Twitter handle because it sounds good, but I can change it to a X handle if needed

Reply:
Your contributions are appreciated. Thank you!